lib/senkyoshi/models/grade_range_percent_criteria.rb

```diff
@@ -0,0 +1,30 @@
+require "senkyoshi/models/grade_range_criteria"
+
+module Senkyoshi
+  class GradeRangePercentCriteria < GradeRangeCriteria
+    def make_completion(mod)
+      super(mod).tap do |completion_requirement|
+        completion_requirement.min_score = get_percentage(
+          @min_score,
+          @points_possible,
+        )
+      end
+    end
+
+    def get_points_possible(resources, id)
+      resource = resources.find_by_id(id)
+      resource.points_possible if resource
+    end
+
+    def get_percentage(min_score, points_possible)
+      possible = points_possible.to_f
+      (min_score.to_f / possible) * possible
```

Comment:
Are you sure this is returning the percentage? It looks to me like it is returning the min_score that was passed in as a float.